src/test/java/org/fisco/bcos/channel/test/parallel/precompile/PerformanceDTTest.java

```diff
@@ -286,44 +291,141 @@
+        threadPool.setMaxPoolSize(threadC > 0 ? threadC : 10);
+        threadPool.setQueueCapacity(threadC > 0 ? threadC : 10);
+        threadPool.initialize();
+
+        Credentials credentials = GenCredential.create();
+
+        TransferSignTransactionManager extendedRawTransactionManager =
+                new TransferSignTransactionManager(
+                        null, credentials, BigInteger.ONE, BigInteger.ONE);
+
+        dagTransfer =
+                DagTransfer.load(
+                        dagTransferAddr,
+                        null,
+                        extendedRawTransactionManager,
+                        new StaticGasProvider(
+                                new BigInteger("30000000"), new BigInteger("30000000")));
+
+        AtomicLong signed = new AtomicLong(0);
+
+        long startTime = System.currentTimeMillis();
+        System.out.println(" => " + dateFormat.format(new Date()));
+
+        for (int i = 0; i < threadC; i++) {
+            threadPool.execute(
+                    new Runnable() {
+                        @Override
+                        public void run() {
+                            while (true) {
+
+                                long index = signed.incrementAndGet();
+                                if (index > totalSignedTxCount.intValue()) {
+                                    break;
+                                }
+                                DagTransferUser from = dagUserMgr.getFrom((int) index);
+                                DagTransferUser to = dagUserMgr.getTo((int) index);
+
+                                Random random = new Random();
+                                int r = random.nextInt(100) + 1;
+                                BigInteger amount = BigInteger.valueOf(r);
+
+                                try {
+                                    String signedTransaction =
+                                            dagTransfer.userTransferSeq(
+                                                    from.getUser(), to.getUser(), amount);
+
+                                    if (index % (totalSignedTxCount.longValue() / 10) == 0) {
+                                        System.out.println(
+                                                "Signed transaction: "
+                                                        + String.valueOf(
+                                                                index
+                                                                        * 100
+                                                                        / totalSignedTxCount
+                                                                                .longValue())
+                                                        + "%");
+                                    }
+                                } catch (Exception e) {
+                                    e.printStackTrace();
+                                    System.exit(-1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.exit() should not be used in J2EE/JEE apps](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4981260)